packages/pangea-sdk/README.md

```diff
@@ -3,25 +3,26 @@
 [![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://pangea.cloud/join-slack/)
 
 
-# Pangea .Net SDK  
+# Pangea .NET SDK
 
-This package helps .Net developers to use Pangea's security services in their applications. In order to use the services, developers need to [register](https://login.aws.us.pangea.cloud/signup) for a Pangea account and obtain an API token.
+This package helps .NET developers to use Pangea's security services in their applications. In order to use the services, developers need to [register](https://login.aws.us.pangea.cloud/signup) for a Pangea account and obtain an API token.
 
 
-## Supported Services  
+## Supported Services
 
-- Pangea Secure Audit Log  
-- Pangea Redact service  
+- Redact
```

Comment:
Hmm quite old this comment. We support AuthN, Embargo, Intel services and FileScan too.

Reply:
I went ahead and just removed this section.